### PR TITLE
banners content updated

### LINKED ISSUE #3555: Update banners content
### Feature description

Banners content need to be updated

### Acceptance criteria

- [x] Updates content of the banner on the top on the qiskit.org page
- [x] Updates content of the banner on the announcement section on the qiskit.org page
- [x] Updates content of the banner on the announcement section on the qiskit.org/learn page
- [x] Updates content of the banner on the top on the qiskit.org/[course] page

### PR DESCRIPTION
**Pull request by aitanaviudes**
## Changes

Closes #3555

- Content of the notifications bar on the top of the qiskit.org page updated.
- Content of the banner on the announcement section of the qiskit.org page updated.
- Content of the banner on the announcement section of the qiskit.org/learn page updated.
- Content of the notifications bar on the top of the qiskit.org/course/introduction-course page updated.

### CODE REVIEW

File: pages/learn/index.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
       :features="[
         {
           descriptionParagraphs: [
-            'We are building a new home for the Qiskit Textbook on IBM Quantum, and we will be migrating courses and tutorials in the coming months.',
+            'We are building a new home for the Qiskit Textbook on IBM Quantum, and we will be migrating courses and tutorials over the coming months. After December 1, 2023, you can continue to access this page’s content on the Qiskit Github.',
```
Comment (y4izus) [inline pages/learn/index.vue:19]: FYI --- This text says "December 1", but the title "Nov 29". I've asked if that is correct and I'm waiting for an answer

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank you @aitanaviudes ! I've some requests before merging :D

File: pages/learn/index.vue (hunk 1 above)
Comment (pandasa123) [reply pages/learn/index.vue]: November 29th is the correct date!

Comment (aitanaviudes) [reply pages/learn/index.vue]: Thank you, Yaiza!

Comment (aitanaviudes) [reply pages/learn/index.vue]: https://github.com/Qiskit/qiskit.org/commit/195a3aa48ff66488057430ba2d2987cce634d442

File: pages/learn/index.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <article class="learn-page">
     <OneXPBanner
-      copy="🎉 Qiskit is getting a new learning experience on IBM Quantum!"
+      copy="🎉  Starting on November 29, 2023 Qiskit Learning Resource will only live on IBM Quantum"
```
[the remark this answers is not in the document]
Comment (aitanaviudes) [reply pages/learn/index.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

File: pages/index.vue [hunk 3]
```diff
@@ -49,7 +49,8 @@
       img-src="/images/one-xp/documentation-screenshot.png"
     >
       <template #title>
-        Qiskit is getting a new documentation<br />and learning experience!
+        Qiskit documentation and learning resources are<br />moving to IBM
```
[the remark this answers is not in the document]
Comment (aitanaviudes) [reply pages/index.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

File: pages/index.vue [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <template>
   <article>
     <OneXPBanner
-      copy="🎉 Qiskit is getting a new documentation and learning experience on IBM Quantum!"
+      copy="🎉  Starting on November 29, 2023 Qiskit Documentation and Learning will only be live on IBM Quantum"
```
[the remark this answers is not in the document]
Comment (aitanaviudes) [reply pages/index.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

File: components/LearnCourse/LearnCourse.vue [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <template>
   <article class="learn-course">
     <OneXPBanner
-      copy="🎉 Qiskit is getting a new learning experience on IBM Quantum!"
+      copy="🎉  Starting on November 29, 2023 Qiskit Learning Resource will only live on IBM Quantum"
```
[the remark this answers is not in the document]
Comment (aitanaviudes) [reply components/LearnCourse/LearnCourse.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

**Pull request merged**